src/Mapbender/WmsBundle/Controller/RepositoryController.php

```diff
@@ -496,9 +495,7 @@ public function instanceLayerPriorityAction($slug, $instanceId, $instLayerId)
             $em->persist($inst);
         }
         $em->flush();
-        $wmsinstance = $this->getDoctrine()
-            ->getRepository("MapbenderCoreBundle:SourceInstance")
-            ->find($instanceId);
+        $wmsinstance = $this->loadEntityByPk("MapbenderCoreBundle:SourceInstance", $instanceId);
```

Comment:
Camelcase?

Reply:
Yes. Kept variable naming as before to minimize diff, but camel-casing would be better.